OpenRA.Mods.Common/Widgets/Logic/Ingame/IngameChatLogic.cs

```diff
@@ -260,7 +256,7 @@ public void CloseChat()
 			Ui.ResetTooltips();
 		}
 
-		public void AddNotificationWrapper(TextNotification notification)
+		public void Handle(TextNotification notification)
```

Comment:
Please implement these interfaces explicitly if possible:
```suggestion
		void INotificationHandler<TextNotification>.Handle(TextNotification notification)
```
(here and the two others below.)
That makes the code more readable by indicating where the `Handle` method belongs to.